src/ray/gcs/gcs_server/gcs_task_manager.h

```diff
@@ -139,17 +163,26 @@ class GcsTaskManager : public rpc::TaskInfoHandler {
   /// a single rpc::TaskEvents entry, as reported by multiple rpc calls from workers.
   ///
   /// When more than `RAY_task_events_max_num_task_in_gcs` task events are stored in the
-  /// the storage, older task events will be replaced by new task events, where older
-  /// task events are approximately task events that arrived in earlier rpc.
+  /// the storage, tasks with lower gc priority as specified by
+  /// `TaskEventGcPolicyInterface` will be evicted first. When new nevents from the
+  /// already evicted task attempts are reported to GCS, those events will also be
```

Comment:
```
When new nevents from already evicted task attempts are reported to GCS, those events will also be dropped.
```

Hmm what does it mean?

Reply:
This means when there are 2 consecutive reports: R1, R2:

R1: T has data loss 
R2: T's some task status changes 
==> GCS should not add R2's T tasks since there's already task data loss.

Reply:
"Any new events for an evicted task attempts will be ignored"?